src/components/layout/nav/Header.stories.tsx

```diff
@@ -33,26 +33,26 @@
                      </div>
                  </Header>
                  <Nav navLevel={NavLevel.SECONDARY} navType={NavType.HEADER}>
-                    <a className="nav-link nav-icon-text submenu-navlink">
+                    <span className="nav-link nav-icon-text submenu-navlink">
```

Comment:
any reason to change a tag with span ?

Reply:
Actually, the anchor tag always requires href tag, In our case we were using <a> for the display text purpose only, that's why, removing them does not impact on functionality

Reply:
ok